app/src/main/res/drawable/quiz_btn.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <solid android:color="#555eec"/>
```

Comment:
Can you add the color hexcode to the `colors.xml` file and then use that color variable? As well as for any other colors you used?